salt/top.sls

```diff
@@ -23,6 +23,8 @@ base:
   'roles:kube-master':
     - match: grain
     - kubernetes-master
+    - flannel
+    - docker
```

Comment:
We will move to an architecture with kubelet in all the machines at some moment, but please open an issue for discussing this before doing a PR because I'm not sure when that will happen...